src/plugins/enzymes/src/DigestSequenceDialog.cpp

```diff
@@ -157,9 +157,11 @@ void DigestSequenceDialog::accept() {
     for (int row = 0; row < itemCount; ++row) {
         const QString &annEncoded = conservedAnnsWidget->item(row)->text();
         QStringList annData = annEncoded.split(" ");
-        assert(annData.size() == 2);
-        QString aName = annData.at(0);
-        QString locationStr(annData.at(1));
+        assert(annData.size() >= 2);
+
+        QString regionString = annData.last();
```

Comment:
You are making the same error here that caused the bug: you are parsing visual representation of data and this process leads to errors. 
First between name and location there may be more spaces (this is a visual formatting only), next the location itself may contain spaces (Genbank format allows it).

What I would suggest as a correct fix: 
1) Store annotation data you need (name & location or the whole annotation data) as QVariant in the QListWidgetItem
2) Take and use it directly with no parsing at all.

Reply:
Fixed